models/models.go

```diff
@@ -17,8 +17,48 @@ var (
 
 // RenderRequest represents a structure for a table render job
 type RenderRequest struct {
-	Title     string `json:"title"`
-	TableType string `json:"type"`
+	Title         string         `json:"title"`
+	Subtitle      string         `json:"subtitle"`
+	Source        string         `json:"source"`
+	TableType     string         `json:"type"`
+	Filename      string         `json:"filename"`
+	Uri           string         `json:"uri"`
+	StyleClass    string         `json:"style_class"`
+	RowFormats    []RowFormat    `json:"row_formats"`
+	ColumnFormats []ColumnFormat `json:"column_formats"`
+	CellFormats   []CellFormat   `json:"cell_formats"`
+	Data          [][]string     `json:"data"`
+	Footnotes     []string       `json:"footnotes"`
+}
```

Comment:
^^^ Do any of these need to include the omit empty tag? Just thinking of cases where we don't send all of the values.